manifest.json

```diff
@@ -4,13 +4,19 @@
   "description": " Get lyrics of currently playing Spotify song so you don't sing along with the wrong ones and embarrass yourself later.",
   "version": "1.0",
   "permissions": [
-    "notifications"
+    "notifications",
+    "nativeMessaging"
   ],
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAt9Y+BeoMhaxYrN3UaguP5yORe3VzxBXShWDqtDh+/O2H+0OmiHM3e55DJlk8A6aXP7jMWhYWBht+bO5p/0RDX21BXs/PX2RLYD2TEonAkTSHjVMHCRgdHgD9zLi4jEQpunGtEWZsSdZYE2tyeC2HfSkkfY0Uo5AYgBEG5dOL7rAw/A1yE/HHOc7HPJQ6s32FWDA85VCbUzYxUguIHqrQW3wNwLugLmqkJ3shiOgRvMubKVaxMjb8yDN8dj5d2fnQnIvSS5GLpUh2TwiyZOau1X4cXdgsdq9cUbxTvJ8vOwwvlwR1qyBQ4IdT5wABV3GfJfsTH+kiiooyp+ZttOO1hwIDAQAB",
```

Comment:
Where's this from?

Reply:
It's the key generated by Google Chrome when the extension is packed. It is essential to include because when loaded without the key, the key changes randomly and we need a exact key to match it with bridge

Reply:
So it will have to be updated once the extension is published?

Reply:
I don't think there needs to be one! After it is published you can remove the key value, because key won't change randomly then. This is kind of a way to simulate that the extension is on webstore and won't be needed after the publish